app/views/browse/_pod-details.html

```diff
@@ -25,6 +25,7 @@
           <a ng-href="{{rcName | navigateResourceURL : 'ReplicationController' : pod.metadata.namespace}}"
             ><span ng-if="deploymentVersion">#{{deploymentVersion}}</span><span ng-if="!deploymentVersion">{{rcName}}</span></a></span>
         </dd>
+        <owner-references ng-if="!dcName" api-object="pod"></owner-references>
```

Comment:
i dont want to remove the link to the DeploymentConfig, since thats where you can edit the configuration